routes/sign-in/sign-in.controller.spec.js

```diff
@@ -0,0 +1,68 @@
+const request = require('supertest')
+const app = require('../../app.js')
+const cheerio = require('cheerio')
+
+const session = require('supertest-session');
+
+function extractCsrfToken(res) {
```

Comment:
Nice way to do this! Seems like we need this function for many tests so maybe it would make sense to put it in a common file like utils/test.helpers.js or something like that. Doesn't have to be part of this PR though

Reply:
This makes sense. I might as well just add it to this PR.